be/src/olap/merger.cpp

```diff
@@ -41,6 +41,10 @@ Status Merger::merge_rowsets(TabletSharedPtr tablet, ReaderType reader_type,
     reader_params.reader_type = reader_type;
     reader_params.rs_readers = src_rowset_readers;
     reader_params.version = dst_rowset_writer->version();
+    reader_params.delete_predicates = tablet->delete_predicates();
```

Comment:
line 44 set reader_params.delete_predicates = tablet->delete_predicates(); and then add it again? Are there duplicate predicates?

Reply:
forgot to delete this line, thanks for remind.